client/src/pages/Discovery/Discovery.tsx

```diff
@@ -0,0 +1,140 @@
+        {
+            id: 'Prize Amount',
+            label: 'Prize Amount',
+            minWidth: 50,
+            align: 'right',
+        },
+        {
+            id: 'Deadline Date',
+            label: 'Deadline Date',
+            minWidth: 50,
+            align: 'right',
+        },
+        {
+            id: 'More Info',
+            label: 'Contest Page',
+            minWidth: 50,
+            align: 'right',
+        },
+    ];
+
+    const handleChangePage = (event: unknown, newPage: number) => {
+        setPage(newPage);
+    };
+
+    const handleChangeRowsPerPage = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setRowsPerPage(+event.target.value);
+        setPage(0);
+    };
+
+    return (
+        <>
+            <AuthHeader linkTo="/createcontest" btnText="create contest" />
+            <Grid container justify="center" className={classes.grid}>
+                <Container className={classes.tableContainer}>
+                    <Grid item>
+                        <Typography className={classes.typography}>All Open Contests</Typography>
+                    </Grid>
+                </Container>
+                <Paper className={classes.paper}>
+                    <TableContainer>
+                        <Table stickyHeader aria-label="sticky table">
+                            <TableHead>
+                                <TableRow className={classes.tableHead}>
+                                    <TableCell className={classes.tableRow} key={columns[0].id} align={columns[0].align} style={{ minWidth: columns[0].minWidth }}>
+                                        {columns[0].label}
+                                    </TableCell>
+                                    <TableCell className={classes.tableRow} key={columns[1].id} align={columns[1].align} style={{ minWidth: columns[1].minWidth }}>
+                                        {columns[1].label}
+                                    </TableCell>
+                                    <TableCell className={classes.tableRow} key={columns[2].id} align={columns[2].align} style={{ minWidth: columns[2].minWidth }}>
+                                        {columns[2].label}
+                                    </TableCell>
+                                    <TableCell className={classes.tableRow} key={columns[3].id} align={columns[3].align} style={{ minWidth: columns[3].minWidth }}>
+                                        {columns[3].label}
+                                    </TableCell>
+                                    <TableCell className={classes.tableRow} key={columns[4].id} align={columns[4].align} style={{ minWidth: columns[4].minWidth }}>
+                                        {columns[4].label}
+                                    </TableCell>
+                                </TableRow>
+                            </TableHead>
+                            <TableBody>
+                                {contests.map(contest => {
```

Comment:
Quick tip, rather than using a curly brace, `contest => {` , use a round brace `contest => (` for an instant return :)